crates/utils/Cargo.toml

```
@@ -27,5 +29,10 @@ features = [
     "Element",
 ]
 
+[features]
+default = ["serde-serialize"]
+serde-serialize = ["serde", "serde_json"]
```

```suggestion
serde-serialize = ["dep:serde", "dep:serde_json"]
```

This makes sure that there's no feature called `serde` or `serde_json`. It does mean that MSRV for 1.61 (iirc) but 2 versions old at this point so I think it's fine, at least for this crate. For context, Yew also bumped the MSRV to 1.61 for this feature

I'd also rename the feature to `serde` since we deserialize as well